deploy/2-deploy-redstone-oracle.ts

```diff
@@ -40,9 +38,6 @@ const func: DeployFunction = async function ({ getNamedAccounts, deployments, ne
   }
 };
 
-func.skip = async ({ network }: HardhatRuntimeEnvironment) =>
-  !["hardhat", "bscmainnet", "bsctestnet", "sepolia", "ethereum", "arbitrumone", "arbitrumsepolia"].includes(
-    network.name,
-  );
 func.tags = ["deploy-redstone"];
+func.skip = async env => !env.network.live;
```

Comment:
We were deploying the RedStone oracle to hardhat in the previous version. Don't we want to deploy it now?

Reply:
yes it would be good deploy it on hardhat.

Reply:
This would be good to run on hardhat as the MockOracle will be used